specs/toolsets/adr_base/setup-toolset.sh

```diff
@@ -58,43 +43,30 @@ case $HAM_OS in
         fi
         ;;
     OSX)
-        export ADR_NDK_PREBUILT=darwin-x86_64
```

Comment:
We need to remove the `NT)` section above since that won't work anymore and its too old anyway.